install-cli.nix

```
@@ -49,7 +52,7 @@ let
         {
           boot.loader.efi.canTouchEfiVariables = lib.mkVMOverride writeEfiBootEntries;
           boot.loader.grub.devices = lib.mkVMOverride diskoSystem.config.boot.loader.grub.devices;
-        }
+        } // (builtins.fromJSON extraSystemConfig)
```

This makes conflict error messages nicer:

```suggestion
          imports = [
             { _file = "disko-install --system-config"} // (builtins.fromJSON extraSystemConfig)
           ];
        }
```